lib/util/device-service-details.ts

```diff
@@ -87,7 +87,15 @@ function getSingleInstallSummary(
 		};
 	}
 
-	const result = {
+	const result: (
+		| (CurrentService &
+				Partial<
+					Pick<ImageInstall, 'installs__image' | 'is_provided_by__release'>
+				>)
+		| CurrentGatewayDownload
+	) &
+		Partial<Pick<ImageInstall, 'image'>> &
+		WithServiceName = {
```

Comment:
All this is b/c now all props that we `delete` must be typed optional.
I don't like it and though of other approaches, but this was implemented this way to optimize this part of code for big fleets.